pkgs/os-specific/linux/kernel/generic.nix

```diff
@@ -104,7 +105,7 @@ let
 
       # Create the config file.
       echo "generating kernel configuration..."
-      echo "$kernelConfig" > "$buildRoot/kernel-config"
+      cp -l "$kernelConfigPath" "$buildRoot/kernel-config"
```

Comment:
Symlinks are less surprising because it is clear, where they came from.

Reply:
Sure thing. Repushed with `ln -s` instead, and retested with sandbox.
```
lrwxrwxrwx 1 nixbld1 nixbld 49 Apr  9 18:30 /tmp/nix-build-linux-config-4.14.32.drv-0/linux-4.14.32/build/kernel-config -> /tmp/nix-build-linux-config-4.14.32.drv-0/.attr-0
```